target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_legacy_mdio.c

```diff
@@ -221,12 +222,18 @@ static int ag71xx_mdio_probe(struct platform_device *pdev)
 	return 0;
 }
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6,11,0)
```

Comment:
probably better to migrate this driver to devm_of_mdiobus_register and remove the _remove function.